speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/ConfigEnums.h

```diff
@@ -73,3 +73,15 @@ typedef struct DataToStore
     bool InterventionData;
     bool MetaData;
 } tDataToStore;
+
+/// @brief The threshold amounts for decisions. floats are values between 0 and 1.
+typedef struct Thresholds
+{
+    float Accel;
+    float Brake;
+    float Steer;
+} tThreshold;
```

Comment:
Just `Threshold` is not a clear name, should be more detailed on what kind of thresholds this contains, like `DecisionThresholds`